File: src/test/java/com/polykhel/billies/service/dto/WalletDTOTest.java
package com.polykhel.billies.service.dto;

import static org.assertj.core.api.Assertions.assertThat;

import com.polykhel.billies.web.rest.TestUtil;
import org.junit.jupiter.api.Test;

class WalletDTOTest {

    @Test
    void dtoEqualsVerifier() throws Exception {
        TestUtil.equalsVerifier(WalletDTO.class);
        WalletDTO walletDTO1 = new WalletDTO();
        walletDTO1.setId(1L);
        WalletDTO walletDTO2 = new WalletDTO();
        assertThat(walletDTO1).isNotEqualTo(walletDTO2);
        walletDTO2.setId(walletDTO1.getId());
        assertThat(walletDTO1).isEqualTo(walletDTO2);
        walletDTO2.setId(2L);
        assertThat(walletDTO1).isNotEqualTo(walletDTO2);
        walletDTO1.setId(null);
        assertThat(walletDTO1).isNotEqualTo(walletDTO2);
    }
}

File: src/test/kotlin/com/polykhel/billnext/service/dto/WalletDTOTest.kt
package com.polykhel.billnext.service.dto

import com.polykhel.billnext.web.rest.equalsVerifier
import org.assertj.core.api.Assertions.assertThat
import org.junit.jupiter.api.Test

class WalletDTOTest {

    @Test
    fun dtoEqualsVerifier() {
        equalsVerifier(WalletDTO::class)
        val walletDTO1 = WalletDTO()
        walletDTO1.id = 1L
        val walletDTO2 = WalletDTO()
        assertThat(walletDTO1).isNotEqualTo(walletDTO2)
        walletDTO2.id = walletDTO1.id
        assertThat(walletDTO1).isEqualTo(walletDTO2)
        walletDTO2.id = 2L
        assertThat(walletDTO1).isNotEqualTo(walletDTO2)
        walletDTO1.id = null
        assertThat(walletDTO1).isNotEqualTo(walletDTO2)
    }
}
